container/incremental_load.sh.tpl

```diff
@@ -115,7 +115,7 @@ function find_diffbase() {
     NEW_DIFF_IDS+=("${diff_id}")
   done
 
-  TOTAL_DIFF_IDS=($(cat "${name}" | python -mjson.tool | \
+  TOTAL_DIFF_IDS=($(cat "${name}" | python3 -mjson.tool | \
```

Comment:
For backward-compatibility of old systems that don't have python3, can you do this instead?

```suggestion

  export PYTHON="python"
  if command -v python3 &>/dev/null; then
      PYTHON="python3"
  fi

  TOTAL_DIFF_IDS=($(cat "${name}" | $PYTHON -mjson.tool | \
```

Reply:
Sounds good to me !
At first I was thinking if there is a way to pull this info directly from bazel, but I'm not very skilled in it and didn't find a way.

Reply:
rules_docker doesn't manage Python toolchain on its own and depends on system Python instead. So this is the only way.